src/main/java/com/minecolonies/coremod/colony/permissions/Permissions.java

```diff
@@ -539,7 +539,7 @@ public void savePermissions(@NotNull final NBTTagCompound compound)
     public Set<UUID> getMessagePlayers()
     {
         return players.values().stream().filter(player ->
-                                                  hasPermission(player.rank, Action.RECEIVE_MESSAGES)).map(player -> player.id).collect(Collectors.toSet());
+                                                  hasPermission(player.getRank(), Action.RECEIVE_MESSAGES)).map(player -> player.getID()).collect(Collectors.toSet());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1612)